mods/cnc/chrome/assetbrowser.yaml

```diff
@@ -1,9 +1,9 @@
 Container@ASSETBROWSER_PANEL:
 	Logic: AssetBrowserLogic
 	X: (WINDOW_RIGHT - WIDTH) / 2
-	Y: (WINDOW_BOTTOM - HEIGHT) / 2
-	Width: 695
-	Height: 435
+	Y: (WINDOW_BOTTOM - 560) / 2
```

Comment:
The other TD panels were changed to use `WINDOW_BOTTOM - HEIGHT` - why change this one away from that?

Reply:
Good catch. This occurred to me as well. There is also an issue with the buttons at the bottom of panels as in some cases they are included in the panel height and in others not. I will look at what works best here and try to tie it all together.

Reply:
My original idea wrt the y alignment was that the panel should be centered, ignoring the title on the top or the buttons below. We don't necessarily need to keep that provided we are consistent.